pkg/capabilities/consensus/ocr3/ocr3_consensus-schema.json

```diff
@@ -5,25 +5,28 @@
     "SignedReport": {
       "type": "object",
       "properties": {
-        "WorkflowExecutionID": {
-          "type": "string"
+        "Report": {
+          "type": "string",
+          "contentEncoding": "base64"
```

Comment:
We still need to figure out what to do longer term so that we can use this type for validation, otherwise it'll expect strings instead of the byte array.